server/schedulers/balance_plan.go

```diff
@@ -117,26 +117,23 @@ func BalancePlanSummary(plans []plan.Plan) (map[uint64]plan.Status, bool, error)
 	for _, pi := range plans {
```

Comment:
```suggestion
	for _, plan:= range plans {
```

Reply:
This is to avoid conflict with plan package